pkg/cookies/csrf.go

```diff
@@ -73,15 +73,23 @@ func NewCSRF(opts *options.Cookie, codeVerifier string) (CSRF, error) {
 
 // LoadCSRFCookie loads a CSRF object from a request's CSRF cookie
 func LoadCSRFCookie(req *http.Request, opts *options.Cookie) (CSRF, error) {
```

Comment:
The code looks good but I would like to use this opportunity to add a few test cases for `LoadCSRFCookie(...)`

I would imaging 4 test cases:

1. No cookie / expected failure
2. One valid cookie
3. One invalid cookie
4. Two cookies: One random cookie and one valid cookie

Reply:
Checkout the test cases for `SetCookie` and `ClearCookie` on how to deal with mocking requests / http communication.

Reply:
> The code looks good but I would like to use this opportunity to add a few test cases for `LoadCSRFCookie(...)`
> 
> 
> 
> I would imaging 4 test cases:
> 
> 
> 
> 1. No cookie / expected failure
> 
> 2. One valid cookie
> 
> 3. One invalid cookie
> 
> 4. Two cookies: One random cookie and one valid cookie

Good idea, I will write tests for this function.